com.unity.netcode.gameobjects/Tests/Runtime/Metrics/PacketLossMetricsTests.cs

```diff
@@ -55,6 +55,9 @@ public IEnumerator TrackPacketLossAsServer()
         }
 
         [UnityTest]
+#if UTP_TRANSPORT_2_0_ABOVE
+        [Ignore("Pending adjustment for UTP 2.0")]
```

Comment:
Can you explain this, was it failing? What's pending? Is there a ticket for this?

Reply:
>JS Fauteux 
>Parce que les tests sont pas fait pour run sur UTP 2.0 and above :’)
(paraphrasing) Because the tests are not made to run on UTP 2.0 and above

Yes, it was failing. 
>Found no matching values for metric of type 'Gauge', with name 'PacketLoss' during '60' frames.

There's no ticket for this yet, but I can create one. I'm not sure which project/tag/etc. to use. Can you elaborate where you'd like it created?

Reply:
In Jira please, just assign it to NETC Team = Tools with your best description of the issue. I'm concerned about this response though, is packet loss not expected to work in UTP 2.0? @Rosme why wouldn't the tests be expected to pass?

Reply:
https://jira.unity3d.com/browse/MTT-4535

I think the comments were all address, will ask re-review. Please let me know if I missed any bits. Thanks!

Reply:
@becksebenius-unity The way to set simulator values has changed in UTP 2.0, which means the way it's set in the tests won't work.